app/services/katello/pulp3/content_view_version/import.rb

```diff
@@ -46,6 +46,30 @@ def destroy_importer(importer_href)
         def self.check!(content_view:, metadata:, path:)
           ImportValidator.new(content_view: content_view, metadata: metadata, path: path).check!
         end
+
+        def self.reset_content_view_repositories_from_metadata!(content_view:, metadata:)
```

Comment:
This method could benefit from a comment explaining why this is necessary

Reply:
updated